qa/suites/fs/multiclient/tasks/ior-shared-file.yaml

```diff
@@ -2,16 +2,24 @@
 tasks:
 - pexec:
     clients:
+      - set -x
       - cd $TESTDIR
-      - wget http://download.ceph.com/qa/ior.tbz2
-      - tar xvfj ior.tbz2
-      - cd ior
+      # partially or incorrectly installed mpich will create a mess and the
+      # configure script or the build process (which is initiated using "make"
+      # command) for the ior project will fail
+      - sudo apt purge -y mpich
```

Comment:
Why would the pkg be partially installed?

Reply:
if there are issues, then the pkg should not be installed at all and then the test would fail due to a different reason.

Reply:
> That said how about `sudo apt install --reinstall mpich`?

reinstall is ambiguous: what does it do -- remove or purge? Plus, different man pages for `apt` command vary whether it is `--reinstall` or subcommand `reinstall`. Therefore, let's keep the current form.

Reply:
Copying  my reply from IRC
> Why would the pkg be partially installed?

I am not sure why this packages is partially/incorrectly installed but I am sure re-installing it fixes the issue. Maybe an older version is installed by default.

Reply:
it does a remove + install, so if you want to remove the pkg along with system wide config files then reinstall might not serve that purpose

Reply:
> Maybe an older version is installed by default.

in that case let us refresh the repository lists before commencing the test cases execution?

Reply:
> it does a remove + install, so if you want to remove the pkg along with system wide config files then reinstall might not serve that purpose

Purging is better so that in future someone doesn't have to debug and find out that some mpich-related config file didn't prevent CephFS job from passing.